### PR TITLE
Add documentation on how to detect screengrab at runtime.

### PR DESCRIPTION
**Pull request by dustinrwh**
Add Advanced screengrab section to screengrab.md for both launch arguments and detecting screengrab at runtime.

Worked with @joshdholtz on figuring out a way to [detect screengrab at runtime](https://github.com/fastlane/fastlane/issues/10863). Figured this would be useful to others as well 👍 

Created new PR because git email was setup incorrectly in other PR.

### CODE REVIEW

**Comment by dustinrwh** [comment]
FYI, error found in CircleCI is from a previous contribution which was already in the docs.

```
[18:26:59]: Found errors in the documentation, more information above
[18:26:59]: Error found in code sample:
launch_arguments([
  "username hjanuschka",
  "build_number 201"
])
Could not find method or action named 'launch_arguments'
```

File: docs/actions/screengrab.md [hunk 1]
```diff
@@ -152,6 +152,91 @@ If you're having trouble getting your device unlocked and the screen activated t
...
+<details>
+<summary>Launch Arguments</summary>
+
+You can provide additional arguments to your testcases on launch. These strings will be available in your tests through `InstrumentationRegistry.getArguments()`.
+
+```ruby
+launch_arguments([
+  "username hjanuschka",
+  "build_number 201"
+])
```
Comment (joshdholtz) [inline docs/actions/screengrab.md]: @dustinrwh Try changing this too...
```rb
supply(
  launch_arguments: {
    "username hjanuschka",
    "build_number 201"
  }
)
```

The tests try to find the action name so let's try this ^

Comment (dustinrwh) [reply docs/actions/screengrab.md]: Sounds good

**Comment by joshdholtz** [comment]
@dustinrwh Hey! Sorry about just getting back to this. Things go lost in the shuffle 😊 This file is now an automatically generated file in this repo and is based off of https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/docs/capture_android_screenshots.md in the main fastlane repo. 

Do you mind making the changes over there? 😊

**Comment by janpio** [comment]
I moved the changes of the generated file to its own PR over at https://github.com/fastlane/fastlane/pull/14303

Are you ok with making these changes this way @dustinrwh? Then I can go ahead an merge both. Thanks.

**Comment by dustinrwh** [comment]
Sounds good to me

**Comment by janpio** [comment]
restarted the tests:

```
[09:16:50]: Found errors in the documentation, more information above
[09:16:50]: Error found in code sample:
match(type: "appstore", app_identifier: "tools.fastlane.app")
'app_identifier' value must be a Array! Found String instead.
[09:16:50]: Error found in code sample:
push_to_git_remote(
  remote: "origin",         # optional, default: "origin"
  local_branch: "develop",  # optional, aliased by "branch", default: "master"
  remote_branch: "develop", # optional, default is set to local_branch
  force: true,    # optional, default: false
  tags: false     # optional, default: true
)
'force' value must be a Fastlane::Boolean! Found TrueClass instead.
```

none of those are from this PR I think, so rebasing would probably fix those.

**Comment by rogerluan** [comment]
Closing this stale PR because all of its content was already merged into the docs as part of https://github.com/fastlane/fastlane/pull/14303

**Pull request closed without merging**